pkgs/by-name/te/textlsp/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  python3,
+  fetchPypi,
+  lib,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "textlsp";
+  version = "0.3.2";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Dn3Jw7B3IjwJLivPtGeWJT1+6VV1UysB8T4PSPaqI1U=";
+  };
+
+  build-system = [ python3.pkgs.setuptools ];
+  dependencies = with python3.pkgs; [
+    pygls
+    lsprotocol
+    language-tool-python
+    tree-sitter_0_21
+    gitpython
+    appdirs
+    openai
+    sortedcontainers
+    langdetect
+    ollama
+  ];
+
+  meta = {
+    description = "Language server for text spell and grammar check with various tools";
+    homepage = "https://github.com/hangyav/textLSP/tree/main";
```

Comment:
add changelog?

Reply:
seemingly no changelog in the repo

Reply:
@justDeeevin https://github.com/hangyav/textLSP/releases/tag/v0.3.2/CHANGELOG.md